x/wasm/keeper/keeper.go

```diff
@@ -154,16 +154,30 @@ func (k Keeper) create(ctx sdk.Context, creator sdk.AccAddress, wasmCode []byte,
 	}
 
 	ctx.GasMeter().ConsumeGas(k.gasRegister.CompileCosts(len(wasmCode)), "Compiling wasm bytecode")
-	checksum, err = k.wasmVM.StoreCode(wasmCode)
-	if err != nil {
-		return 0, checksum, sdkerrors.Wrap(types.ErrCreateFailed, err.Error())
+	isSimulation, ok := types.ExecModeSimulation(ctx)
+	if !ok {
+		k.Logger(ctx).Info("cannot getexec mode from context")
```

Comment:
Should this be a warning instead? If this case happens, your nodes can still be attacked.

Reply:
Oh, I see. Alright, in that case info probably fits the best.